### ISSUE TITLE
[TextInputEditText] Text get hidden when selected from Google Saved passwords

### ISSUE THREAD

**Issue by KousalyaRaj** [material-components/material-components-android#2037]
First of all, when I choose username and password TextInputEditText from google saved passwords by Autofill. Background color of both edittext got yellow color shade. I fixed that by adding below line in v26/styles.xml ,

`<item name="android:autofilledHighlight">@drawable/round_button</item> `

But after that got another issue. Now, the text itself hidden when choosing from google password. And Right drawable of that field also hidden. It is visible only after typing any text to that. My cursor color changed from white to blue.

After choosing username and password from google saved credentials, my TextInputEditText fields text color need to be white. And Right drawable need to be displayed. How to achieve that.

`<com.google.android.material.textfield.TextInputLayout
            android:id="@+id/usernameLayout"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:layout_marginTop="10dp"
            android:layout_marginBottom="15dp"
            app:hintEnabled="false">

            <com.google.android.material.textfield.TextInputEditText
                android:id="@+id/login_username_input"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:layout_weight="0.0"
                android:drawableEnd="@drawable/user_img"
                android:background="@drawable/round_button"
                android:hint="@string/login_username_hint"
                android:imeActionLabel="@string/action_next"
                android:imeOptions="actionNext"
                android:inputType="textUri"
                android:importantForAutofill="yes"
                android:maxLines="1"
                android:autofillHints="@string/login_username_hint"
                android:nextFocusDown="@+id/login_password_input"
                android:paddingStart="10dp"
                android:paddingLeft="10dp"
                android:text=""
                android:textColor="#FFFFFF"
                android:textColorHint="#FFFFFF"
                android:textCursorDrawable="@null"
                tools:ignore="RtlSymmetry" />
        </com.google.android.material.textfield.TextInputLayout>

        <com.google.android.material.textfield.TextInputLayout
            android:id="@+id/passwordLayout"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:layout_below="@id/usernameLayout"
            android:layout_marginTop="10dp"
            android:layout_marginBottom="15dp"
            app:hintEnabled="false"
            app:passwordToggleEnabled="false"
            app:passwordToggleTint="@color/white">

            <com.google.android.material.textfield.TextInputEditText
                android:id="@+id/login_password_input"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:layout_below="@id/login_username_input"
                android:layout_weight="0.0"
                android:drawableEnd="@drawable/password"
                android:background="@drawable/round_button"
                android:hint="@string/login_password_hint"
                android:imeActionLabel="@string/action_done"
                android:imeOptions="actionDone"
                android:inputType="textPassword"
                android:maxLines="1"
                android:nextFocusDown="@+id/login_submit_btn"
                android:paddingStart="10dp"
                android:paddingLeft="10dp"
                android:text=""
                android:textColor="#FFFFFF"
                android:textColorHint="#FFFFFF"
                android:textCursorDrawable="@null"
                tools:ignore="RtlSymmetry" />
        </com.google.android.material.textfield.TextInputLayout>`

I am using below Material Android Library version 

`implementation 'com.google.android.material:material:1.2.1'`

Tested this issue in Android 10.

**Comment by KousalyaRaj** [comment]
I solved this issue by adding below line in v26/styles.xml ,
`<item name="android:autofilledHighlight">@color/transparent</item>`
